ibtk/src/boundary/HierarchyGhostCellInterpolation.cpp

```diff
@@ -517,11 +503,9 @@ HierarchyGhostCellInterpolation::resetTransactionComponents(
         const std::vector<RobinBcCoefStrategy<NDIM>*>& robin_bc_coefs = d_transaction_comps[comp_idx].d_robin_bc_coefs;
 #if !defined(NDEBUG)
         bool null_bc_coefs = true;
-        for (std::vector<RobinBcCoefStrategy<NDIM>*>::const_iterator cit = robin_bc_coefs.begin();
-             cit != robin_bc_coefs.end();
-             ++cit)
+        for (const auto robin_bc_coef : robin_bc_coefs)
```

Comment:
In range-based for loops, shouldn't this always be either `const auto&&` (if we are not changing the contents) or `auto&&` (if we are)?